src/ufoLib2/objects/info.py

```diff
@@ -279,9 +297,63 @@ def _validate_weight_class(self, attribute: Any, value: Optional[int]) -> None:
     year: Optional[int] = None
 
     @classmethod
-    def read(cls, reader: UFOReader) -> "Info":
+    def read(cls, reader: UFOReader, lib: Dict[str, Any]) -> "Info":
```

Comment:
i think Info.read method should have that lib parameter as optional
and get it from the reader if one is not provided by caller